release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -717,6 +717,157 @@ In {product-title} 4.7, _Cluster Logging_ became _Red Hat OpenShift Logging_. Fo
+
+* Alertmanager pods (ports 9092 and 9097)
+* kube-state-metrics pod (ports 8443 and 9443)
+* openshift-state-metrics pod (ports 8443 and 9443)
+* node-exporter pods (port 9100)
+* Grafana pod (port 3002)
+* prometheus-adapter pods (port 6443)
+* prometheus-k8s pods (ports 9092 and 10902)
+* Thanos query pods (ports 9092, 9093 and 9094)
+* Prometheus Operator (ports 8080 and 8443)
+* telemeter-client pod (port 8443)
+
+If you have enabled user-defined monitoring, the following pods now use the profile:
+
+* prometheus-user-workload pods (ports 9091 and 10902)
+* prometheus-operator pod (ports 8080 and 8443)
+
+[id="ocp-4-10-monitoring-changes-to-alerting-rules"]
+==== Changes to alerting rules
+
+* *New*
+** Added a `namespace` label to all Thanos alerting rules.
+** Added the `openshift_io_alert_source="platform"` label to all platform alerts. 
+
+
+* *Changed*
+** Renamed `AggregatedAPIDown` to `KubeAggregatedAPIDown`.
+** Renamed `AggregatedAPIErrors` to `KubeAggregatedAPIErrors`.
+** Removed the `HighlyAvailableWorkloadIncorrectlySpread` alert.
+** Improved the description of the `KubeMemoryOvercommit` alert.
+** Improved `NodeFilesystemSpaceFillingUp` alerts to make it consistent with the Kubernetes garbage collection thresholds.
+** Excluded `ReadOnlyMany` volumes from the `KubePersistentVolumeFillingUp` alerts.
+** Extended `PrometheusOperator` alerts to include the Prometheus operator running in the `openshift-user-workload-monitoring` namespace.
+** Replaced the `ThanosSidecarPrometheusDown` and `ThanosSidecarUnhealthy` alerts by `ThanosSidecarNoConnectionToStartedPrometheus`.
+** Changed the severity of `KubeletTooManyPods` from `warning` to `info`.
+** Enabled exclusion of specific persistent volumes from `KubePersistentVolumeFillingUp` alerts by adding the `alerts.k8s.io/KubePersistentVolumeFillingUp: disabled` label to a persistent volume resource.
+
+
+[NOTE]
+=====
+Red Hat does not guarantee backward compatibility for recording rules or alerting rules.
+=====
+
+[id="ocp-4-10-monitoring-changes-to-metrics"]
+==== Changes to metrics
+
+* Pod-centric cAdvisor metrics available at the slice level have been dropped.
+* The following metrics are now exposed:
+** `kube_poddisruptionbudget_labels`
+** `kube_persistentvolumeclaim_labels`
+** `kube_persistentvolume_labels`
+* Metrics with the name `kube_*annotation` have been removed from `kube-state-metrics`.
+
+[NOTE]
+=====
+Red Hat does not guarantee backward compatibility for metrics.
+=====
+
+[id="ocp-4-10-monitoring-added-hard-anti-affinity-rules-and-poddistruptionbudgets"]
+==== Added hard anti-affinity rules and poddisruptionbudgets for certain components
+
+With this release, hard anti-affinity rules and `poddisruptionbudgets` have been enabled for the following monitoring components to reduce downtime during patch upgrades:
+
+* Alertmanager
++
+[NOTE]
+=====
+As part of this change, the number of Alertmanager replicas has been reduced from three to two.
```

Comment:
should we link to "known issues" or state that if you've configured persistent storage for Alertmanager, you want to read the "known issues"?

Reply:
I'll point them to the Known Issues. We can't use internal cross-reference links.

Reply:
fixed